web/source/dom/preProcessor.ts

```diff
@@ -157,6 +157,19 @@ namespace com.keyman.dom {
        */
       s.Lmodifiers |= (e.metaKey ? modifierCodes['META']: 0);
 
+      // Physically-typed keys require use of a 'desktop' form factor and thus are based on a virtual "physical" Device.
+      s.device = keyman.util.physicalDevice.coreSpec;
+
+      // Check for any browser-based keymapping before proceeding further.
+      // This is needed _now_ for mnenomic keyboards.  (See https://github.com/keymanapp/keyman/issues/1125.)
+      if(!keyman.isEmbedded && s.device.browser == utils.Browser.Firefox) {
+        // I1466 - Convert the - keycode on mnemonic as well as positional layouts
```

Comment:
This comment is now outdated.

Reply:
True, but oddly still very appropriate - that's literally one thing this PR is fixing.

The new comments (L163-164) are a more general version of that, so yeah, no need for the redundancy.  Will delete. :+1: